### DISCUSSION TITLE
Sync mariaDB to Mysql possible?

### DISCUSSION THREAD

**Discussion by kaktux** [welaika/wordmove#658] [Q&A]
Hi there,

maybe a quite basic question - but im not sure if its causing an error.
A friend of mine asked to use my wordmove virtual machine (ubuntu 22.04) for his website.
So we simply cloned the whole vm and added a blank wordpress installation + movefile - to have a working "syncing machine".

What we didnt think about: my hoster uses mariadb (v10; so does the vm/local environment), his mysql (v5).

When i want to sync the local environment to the mysql host i get:

>  wordmove push --all -e live
> Traceback (most recent call last):
>         15: from /home/sysadmin/.rbenv/versions/2.7.6/bin/wordmove:23:in `<main>'
>         14: from /home/sysadmin/.rbenv/versions/2.7.6/bin/wordmove:23:in `load'
>         13: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/exe/wordmove:6:in `<top (required)>'
>         12: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
>         11: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
>         10: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
>          9: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
>          8: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/lib/wordmove/cli.rb:105:in `push'
>          7: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/lib/wordmove/deployer/base.rb:11:in `deployer_for'
>          6: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/lib/wordmove/movefile.rb:34:in `load_dotenv'
>          5: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/lib/wordmove/movefile.rb:46:in `environment'
>          4: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/wordmove-5.2.2/lib/wordmove/movefile.rb:30:in `fetch'
>          3: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/2.7.0/psych.rb:349:in `safe_load'
>          2: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/2.7.0/psych.rb:390:in `parse'
>          1: from /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/2.7.0/psych.rb:456:in `parse_stream'
> /home/sysadmin/.rbenv/versions/2.7.6/lib/ruby/2.7.0/psych.rb:456:in `parse': (<unknown>): did not find expected key while parsing a block mapping at line 46 column 5 (Psych::SyntaxError)

Is that a database error? I'm unsure if i can simply use wordmove to sync a mariadb installation/db to a mysql database?  Or if the error is caused by something different.

**Comment by alessandro-fazzi** [comment]
Hello @kaktux ,

that seems a typo in your `movefile.yml`.

> line 46 column 5

is referred to that file by any chance.

Push/pull-ing between different DB is a bit a matter of faith IMO, but you could be lucky if the two are inter-compatible. But your current error looks like another matter.
